acceptance-tests/mysql_test.go

```diff
@@ -86,4 +89,76 @@ var _ = Describe("MySQL", Label("mysql"), func() {
 		Expect(string(b)).To(ContainSubstring("error connecting to database: failed to verify the connection"), "force TLS is enabled by default")
 		Expect(string(b)).To(ContainSubstring("Error 1045 (28000): Access denied for user"), "postgresql client cannot connect to the postgres server due to invalid TLS")
 	})
+
+	// As we introduce the 'use_managed_admin_password' feature, some users may wish to update existing DBs.
+	// This is a tactical test that should exist for this changeover period and is not intended to be a forever test.
+	// Due to limitations in Tofu/AWS provider/AWS the operation to switch fails first time, then succeeds on
+	// a second attempt. That's not an ideal customer experience, and this test exists to ensure that what we
+	// document works, and make us aware if the behavior changes.
+	It("allows 'use_managed_admin_password' to be enabled", Label("managed-password"), func() {
+		By("creating a service instance")
+		serviceInstance := services.CreateInstance("csb-aws-mysql", services.WithPlan("default"))
+		defer serviceInstance.Delete()
+
+		By("pushing an unstarted app")
+		appManifest := jdbcapp.ManifestFor(jdbcapp.MySQL)
+		app := apps.Push(apps.WithApp(apps.JDBCTestAppMysql), apps.WithManifest(appManifest))
+		defer apps.Delete(app)
+
+		By("binding the app to the service instance")
+		binding := serviceInstance.Bind(app)
+
+		By("starting the app")
+		apps.Start(app)
+
+		By("creating an entry using the app")
+		value := random.Hexadecimal()
+		var userIn jdbcapp.AppResponseUser
+		app.POST("", "?name=%s", value).ParseInto(&userIn)
+
+		By("updating the service to set 'use_managed_admin_password' a first time which is expected to fail")
+		params := `{"use_managed_admin_password": true}`
+		session := cf.Start("update-service", serviceInstance.Name, "-c", params, "--wait")
+		Eventually(session).WithTimeout(time.Hour).Should(gexec.Exit(), func() string {
```

Comment:
The return code is certainly non-zero, but I don't know what number it is. Probably it is `1`. I'm happy to make that change as it does make the test more specific. Ideally I'd like a `ExitWithNonZero()` matcher because the actual number doesn't matter.